types/gulp-modernizr/index.d.ts

```diff
@@ -0,0 +1,79 @@
+// Type definitions for gulp-modernizr 1.0
+// Project: https://github.com/doctyper/gulp-modernizr
+// Definitions by: remisery <https://github.com/remisery>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/// <reference types="node" />
+
+import stream = require('stream');
+
+declare function GulpModernizr(parames?: GulpModernizr.Params): stream.Transform;
+declare function GulpModernizr(file?: string, parames?: GulpModernizr.Params): stream.Transform;
+
+declare namespace GulpModernizr {
+    interface Params {
+        /**
+         * Avoid unnecessary builds (see Caching section below)
+         */
+        cache?: boolean;
+
+        /**
+         * Path to the build you're using for development.
+         */
+        devFile?: string;
```

Comment:
You want this and dest to be string | false instead since you can set this as false